packages/ui/src/components/ChainIcon/index.tsx

```diff
@@ -25,7 +25,7 @@ import osmosisIconLight from 'shared-utils/assets/icons/osmosis-light.svg?url';
 import persistenceIconDark from 'shared-utils/assets/icons/persistence-dark.svg?url';
 import provenanceIconDark from 'shared-utils/assets/icons/provenance-dark.svg?url';
 import quasarIconDark from 'shared-utils/assets/icons/quasar-dark.png';
-import quicksliverIconLight from 'shared-utils/assets/icons/quicksilver-light.svg?url';
```

Comment:
did you change this intentionally?

Reply:
that icon was actually the same as `quicksilverLogoLight` (icon with the quicksilver name), so I downloaded the svg icon (plain logo without the network name) from figma and imported it here
@teamchong